testapps/php71_e2e/tests/tests/EndToEndTest.php

```diff
@@ -69,6 +69,8 @@ public static function setUpBeforeClass()
             );
         }
         self::deploy($project_id, $e2e_test_version);
+        echo 'Sleeping for the deployment settles down' . PHP_EOL;
+        sleep(60);
```

Comment:
I see some cases where there is a failure after few successful requests (502), so currently it's not enough to see a single success.

I'll try to introduce retries on the tests themselves.